src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -118,6 +115,8 @@ public partial class DefaultHubDispatcher<THub> : HubDispatcher<THub> where THub
                     Log.UnsupportedMessageReceived(_logger, hubMessage.GetType().FullName);
                     throw new NotSupportedException($"Received unsupported message: {hubMessage}");
             }
+
+            return Task.CompletedTask;
```

Comment:
nit: What about making PingMessage and CancelInvocationMessage return completed task rather than breaking. Means this return can be removed